code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -466,8 +457,8 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 			icon_state = "wonderprodProbe"
 			item_state = "wonderprodProbe"
 
-/obj/item/abductor_baton/attack(mob/target, mob/living/user)
-	if(!isabductor(user))
+/obj/item/abductor/baton/attack(mob/target, mob/living/user)
+	if(!AbductorCheck(user))
```

Comment:
No change in behavior, just provides a feedback message to the user to why it didn't work if they're not an abductor.